### PR TITLE
[enterprise-4.5] fix toc issue

### PR DESCRIPTION
**Pull request by pneedle-rh**
This applies to branch/enterprise-4.5 only.

This issue relates to https://github.com/openshift/openshift-docs/pull/30712.

This PR:

* Ensures that the `toc::[]` line in all assemblies is preceded by one blank line
* Removes unnecessary attributes from `modules/common-attributes.adoc`

This PR does not update `contributing_to_docs/doc_guidelines.adoc`, because that document is updated in the master branch only.

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please. Thanks!

File: installing/installing_openstack/installing-openstack-load-balancing.adoc [hunk 1]
```diff
@@ -5,4 +5,4 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/installation-osp-balancing-external-loads.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/installation-osp-balancing-external-loads.adoc[leveloffset=+1]
```
Comment (pneedle-rh) [inline installing/installing_openstack/installing-openstack-load-balancing.adoc:8]: Where you see the last line in a file being removed and added in this PR, this is the TOC blank line correction script adding new lines to the end of a file where one did not previously exist. It was a side effect of the script, but it is useful.

Comment (bobfuru) [reply installing/installing_openstack/installing-openstack-load-balancing.adoc:8]: Interesting, thanks for pointing it out!

File: nodes/clusters/nodes-cluster-resource-levels.adoc [hunk 2]
```diff
@@ -3,7 +3,6 @@
 = Estimating the number of pods your {product-title} nodes can hold
 include::modules/common-attributes.adoc[]
 
-
 toc::[]
```
Comment (bobfuru) [inline nodes/clusters/nodes-cluster-resource-levels.adoc:7]: I'll just call it out once as it occurs often but it should not be of concern to us if there are multiple blank lines below `toc::[]`, correct?

**Review by bobfuru — COMMENTED** [review]
One comment about spacing below `toc::[]` but otherwise LGTM!

File: nodes/clusters/nodes-cluster-resource-levels.adoc (hunk 2 above)
Comment (pneedle-rh) [reply nodes/clusters/nodes-cluster-resource-levels.adoc:7]: @bobfuru thank you for reviewing this! I noticed that too, but thought it best to focus for this PR on the blank lines prior to the `toc::[]` declaration in each assembly.

We could use a similar version of the script that I wrote to ensure that there is only one blank line after the `toc::[]` declaration too, in another set of PRs. My understanding is that this should not impact whether the TOCs are built. For now I think the important thing is to make sure that there is one blank line between the header and the TOC declaration in each assembly. This means that we can remove the unnecessary attributes from the `modules/common-attributes.adoc` file and still have the TOCs built successfully.

**Comment by Preeticp** [comment]
I have reviewed the ODC, odo and Pipelines sections, they all look good to me. Thank you for doing this @pneedle-rh

File: nodes/clusters/nodes-cluster-resource-levels.adoc (hunk 2 above)
Comment (bobfuru) [reply nodes/clusters/nodes-cluster-resource-levels.adoc:7]: I agree 💯  percent, @pneedle-rh. Thank you!

**Comment by Preeticp** [comment]
Verified builds/ as well, all good.

**Pull request merged**